UnityProject/Assets/Scripts/Player/PlayerSync.Server.cs

```diff
@@ -264,6 +264,7 @@ public void NotifyPlayer(GameObject recipient, bool noLerp = false)
 	{
 		serverState.NoLerp = noLerp;
 		var msg = PlayerMoveMessage.Send(recipient, gameObject, serverState);
+		PlayerUprightMessage.Sync(recipient, gameObject);
```

Comment:
why not use
https://github.com/unitystation/unitystation/blob/0142d58a829c05cfc825b6dc0162b30c44c7db3c/UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs#L313
method instead?

Reply:
Not sure what you mean by "use"...maybe be more specific about the change you are suggesting. This method gets called from SyncPlayerData. Because this new logic has to do with an individual player's status it made sense to me to put it in playersync. I can't think of any situation where you'd want to sync PlayerSync without the player's upright status so putting it all in this method prevents someone from forgetting to do both.

Reply:
ahh, sorry, didn't notice that it's called inside `NotifyPlayer`, not `NotifyPlayers`